docs/pages/deploy-a-cluster/helm-deployments/kubernetes-cluster.mdx

```diff
@@ -397,6 +403,21 @@ the default one in case there is a problem.
   ```
 </Admonition>
 
+## Troubleshooting
+
+If you are experiencing errors connecting with the Teleport Cluster check the proxy
+and auth pods status. A successful state should be both pods running as below:
+
+```code
+$ kubectl get pods -n teleport-cluster
+NAME                                READY   STATUS    RESTARTS   AGE
+teleport-cluster-auth-5f8587bfd4-p5zv6    1/1     Running   0          48s
+teleport-cluster-proxy-767747dd94-vkxz6   1/1     Running   0          48s
+```
+If a Pod is Pending or not running completely use the `kubectl logs` and `kubectl describe` 
```

Comment:
What does "not running completely" mean? Does Kubernetes have the concept of a pod that is only partially running?

Reply:
Updated language to describe the status or not ready which is the more typical language

Reply:
`0/1` and the Pod is running mean it isn't ready.